docs/integrations/gitops/github.mdx

```diff
@@ -122,8 +129,8 @@ The private key is at the location you chose and the public key is at the same l
     `shell
 curl -X PUT "https://api.permit.io/v2/projects/<your-project-id-or-key>/repos/<your-repo-id-or-key>/activate" -H "Authorization: <your-permit-api-key>"
 `
-12. To add custom policy code for each environment, ensure you have checked out the correct branch using permit/generated/<env_id>. Insert your policy code exclusively within the `custom` folder. This can be done by either editing the `custom/root.rego` file with your custom Rego code or by creating a new file and importing it in the `custom/root.rego` file. 
-Please avoid editing policy code outside of this folder, as it could potentially be overridden by our code or break the Rego code.
+12. To add custom policy code for each environment, ensure you have checked out the correct branch using permit/generated/<env_id>. Insert your policy code exclusively within the `custom` folder. This can be done by either editing the `custom/root.rego` file with your custom Rego code or by creating a new file and importing it in the `custom/root.rego` file.
+    Please avoid editing policy code outside of this folder, as it could potentially be overridden by our code or break the Rego code.
```

Comment:
```suggestion
    _Please avoid editing policy code outside of this folder, as it could potentially be overridden by our code or break the Rego code._
```